.github/workflows/python.yml

```diff
@@ -212,39 +197,43 @@ jobs:
           npm run with-server test:python
 
   macos:
-    runs-on: ${{ matrix.platform.runner }}
+    runs-on: ${{ matrix.runner }}
     strategy:
+      fail-fast: false
       matrix:
-        platform:
-          - runner: macos-12
-            target: x86_64
```

Comment:
ironically i can't get `x86_64` to work now but `aarch64` does

Reply:
What's the error?